Source/g_game.c

```diff
@@ -472,7 +480,12 @@ void G_BuildTiccmd(ticcmd_t* cmd)
     cmd->buttons |= BT_USE;
   }
 
-  if (!novert)
+  // [crispy] mouse look
+  if (mouselook)
+  {
+      cmd->lookdir = /* mouse_y_invert ? -mousey : */ mousey;
```

Comment:
Because `mouseSensitivity_vert` is 0 by default, `mousey` is always 0. Now we have a `novert` switch, so we can change the default value of `mouseSensitivity_vert`? Also is there a plan for `mouse_y_invert`?

Reply:
Sure, we should change the default vetical mouse sensitivity to something non-zero, good point!

Not sure about `mouse_y_invert`. Somebody must have requested this for Crispy once, though I cannot imagine playing with this enabled - makes me seasick. However, it would fit nicely into the "Input" section on the second page of the General menu. :wink:

Reply:
The joystick is "inverted" by default, and I remember the days when inverted controls were the default in old console games. But I agree that this option is not necessary.

Reply:
Added eight years ago, I don't have a clue anymore where this came from 😆 
https://github.com/fabiangreffrath/crispy-doom/commit/706220356f395d09ed0df64c5c7f16d7b74b32b7